components/kyma-environment-broker/internal/process/provisioning/btp_operator_overrides.go

```diff
@@ -0,0 +1,63 @@
+package provisioning
+
+import (
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/ptr"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/sirupsen/logrus"
+)
+
+const BTPOperatorComponentName = "btp-operator"
+
+type BTPOperatorOverridesStep struct {
+	operationManager *process.ProvisionOperationManager
+}
+
+func NewBTPOperatorOverridesStep(os storage.Operations) *BTPOperatorOverridesStep {
```

Comment:
nit: I think `storage.Operations` is not necessary for the provisioning BTP Operator overrides